<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1706</width>
    <height>1339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="plotAngle" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>1800</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotCadence" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>1800</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotVelocity" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>1800</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotGear" native="true">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>600</y>
      <width>1800</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotShiftUp" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>795</y>
      <width>1800</width>
      <height>125</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotShiftDown" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>910</y>
      <width>1800</width>
      <height>110</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1706</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
